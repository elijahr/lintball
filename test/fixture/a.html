<html>

<head><title>
      A
</title></head>

      <body>
    <h1>
  B</h1>
      </body>
