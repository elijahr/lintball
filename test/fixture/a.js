
const a =            {
  a: "b", 1:
  2 }

module.exports = {
       foo: function   () { throw      new Error('foo')},
  bar:
    (

    )    => ({ a   : "b",  1:   2,
  ...a   }) ,
}
