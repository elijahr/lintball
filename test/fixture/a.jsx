/* eslint-disable import/no-unresolved */

import * as ReactDOM from   "react-dom/client";

import React  from 'react';





ReactDOM.render(
                <h1>Hello, world!</h1>,
  document.getElementById('root')
          );
