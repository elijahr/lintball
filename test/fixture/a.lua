type       A = { b: number, c: number }

      local   a: A = { b = 1,
      c = 2 }

print(
        a.b, a.c
      )
