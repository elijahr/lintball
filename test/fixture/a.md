|          aaaa | bbbbbb              |  cc |
| :--- | :-------------------------------: | ------: |
| a |   b |   c |
