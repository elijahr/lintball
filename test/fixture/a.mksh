
a() { echo

}

b() {


echo
}


c=(               "a" "b"            "c")

for var in "${c[@]}"


do
echo "$var"
done
