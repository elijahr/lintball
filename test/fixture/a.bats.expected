#!/usr/bin/env bats

@test 'a' {
  echo
}

@test 'b' {
  echo
}
